src/Common/IntervalKind.cpp

```diff
@@ -15,7 +16,7 @@ Int32 IntervalKind::toAvgSeconds() const
     {
         case IntervalKind::Nanosecond:
         case IntervalKind::Microsecond:
-        case IntervalKind::Millisecond: return 0; /// fractional parts of seconds have 0 seconds
+        case IntervalKind::Millisecond: return 0;
```

Comment:
(nit: return type of toAvgSeconds could have been unsigned but that's okay)

Reply:
I wasn't the one who introduced this so left it as is